scripts/pre_push_hook.py

```diff
@@ -36,7 +36,8 @@
 import subprocess
 import sys
 
-import python_utils
+sys.path.append(os.getcwd())
```

Comment:
Please add a detailed comment to explain why we are doing this.

Reply:
Done